app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,38 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = Recipe.all
```

Comment:
Hi @ulises2607 I think here you should list the recipe of the logged-in user as this is one of the project requirements.
You can use something like
`@recipes = current_user.recipe`
After setting the relation in the user model

Reply:
Thank you for the suggestion partner!! 

https://github.com/ulises2607/rails-recipe-app/assets/97530670/0fe12fc8-383d-4fef-8ad8-6b749cd01eec